### PR TITLE
ScrollingTextView component

### PR DESCRIPTION
**Pull request by ChrisLR**

### CODE REVIEW

File: clubsandwich/ui/scrollingtextview.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+
+class ScrollingTextView(View):
+  """
+  :param int value: Initial value
+  :param func callback: Called with updated values
+  :param int|None min_value: Min value or ``None``
+  :param int|None max_value: Max value or ``None``
+
+  See :py:class:`View` for the rest of the init arguments.
```
Comment (irskep) [inline clubsandwich/ui/scrollingtextview.py]: Docstring needs to be fixed to reflect the actual function signature and class. I'll fix if you don't get to it.

**Comment by irskep** [comment]
FYI, I don't mind people submitting PEP8-compliant code, as long as I don't have to convert the whole codebase.

I think I will consider converting to PEP8 when there are enough regular contributors for me to be out-voted (i.e. 2) 😁

**Review by irskep — CHANGES_REQUESTED** [review]

**Comment by ChrisLR** [comment]
Fixed docstring, by the way, indent of four is far less a pain in the ass with 120 characters per line :)

**Comment by irskep** [comment]
Wow you're quick. :-) The other thing I forgot to mention is, we need to either:

1. Create a list of classes to document for the next release in a GitHub ticket, or
2. Add this class to `docs/ui/components.rst` and make sure it all renders.

I prefer #1 to avoid putting a burden  on everybody submitting new components, so I'll go make that ticket now.

**Comment by irskep** [comment]
>  indent of four is far less a pain in the ass with 120 characters per line

True, but I actually like 80 character wrap! It means I can comfortably split windows in half on a laptop screen.

**Comment by ChrisLR** [comment]
You must have good eyes, mine are crippled after half an hour coding session with a single window and big fonts on a laptop screen. 😄

**Pull request merged**